.golangci.yaml

```diff
@@ -1,3 +1,6 @@
+run:
+  timeout: 9m
```

Comment:
This seems like a really long timeout for such a small repo. Were you actually seeing it take this long, or is this number just a "it-should-never-take-this-long" backstop?

Reply:
I was just related to this PR https://github.com/letsencrypt/pebble/pull/379